### PR TITLE
feat(core): add default revalidate target

### PR DESCRIPTION
**Pull request by jorgemoya**
## What/Why?
Allow an env variable to set a default revalidate target.

## Testing
Locally.

### CODE REVIEW

File: apps/core/client/queries/get-route.ts [hunk 1]
```diff
@@ -28,6 +29,7 @@ export const getRoute = async (path: string) => {
   const response = await client.fetch({
     document: query,
     variables: { path },
+    fetchOptions: { next: { revalidate } },
```
Comment (jorgemoya) [inline apps/core/client/queries/get-route.ts:32]: Maybe we don't want this here?

File: apps/core/client/queries/get-store-status.ts [hunk 2]
```diff
@@ -13,7 +14,7 @@ export const GET_STORE_STATUS_QUERY = /* GraphQL */ `
 
 export const getStoreStatus = async () => {
   const query = graphql(GET_STORE_STATUS_QUERY);
-  const { data } = await client.fetch({ document: query });
+  const { data } = await client.fetch({ document: query, fetchOptions: { next: { revalidate } } });
```
Comment (jorgemoya) [inline apps/core/client/queries/get-store-status.ts:17]: Should we set a sensible limit here, like 3-5 mins?

File: apps/core/client/revalidate-target.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const revalidate = process.env.NEXT_PUBLIC_DEFAULT_REVALIDATE_TARGET
+  ? Number(process.env.NEXT_PUBLIC_DEFAULT_REVALIDATE_TARGET)
+  : undefined;
```
Comment (matthewvolk) [inline apps/core/client/revalidate-target.ts:3]: If we default to 3600 in `.env.example`, should we also default to 3600 here?

```suggestion
  : 3600;
```

**Review by matthewvolk — COMMENTED** [review]
This LGTM! 👍 Before I approve, can you add this default environment variable to https://github.com/bigcommerce/catalyst/blob/main/packages/create-catalyst/src/utils/write-env.ts as well?

File: apps/core/client/revalidate-target.ts (hunk 3 above)
Comment (jorgemoya) [reply apps/core/client/revalidate-target.ts:3]: Good catch, I think that makes sense.

**Review by matthewvolk — APPROVED** [review]
👍👍

**Pull request merged**